### PR TITLE
Conference partners page

### PR DESCRIPTION
**Pull request by lpil**
smile-cops

Blocked by Badger Brain being broken. Bah.

- [x] Title for bronze partners
- [x] Tighter sponsor rows. (don't render empty heading elements)

### CODE REVIEW

File: server/routing-test.js [hunk 1]
```diff
@@ -57,19 +50,22 @@ describe('server routing', () => {
...
 
     it('GET /conference redirects to /', () => {
+      useDummyData();
       return get('/conference')
         .expect(302)
         .expect('LOCATION', '/');
     });
 
     it('GET /community redirects', () => {
+      useDummyData();
```
Comment (zoeabryant) [inline server/routing-test.js]: can `useDummyData();` be stuck in a beforeEach?

Comment (lpil) [inline server/routing-test.js]: Yes but I don't like beforeEach as they're easy to lose track of. I would much rather be explicit.

**Comment by asavin** [comment]
Looks good to me

**Comment by AndrewBestbier** [comment]
screen shot 2016-08-23 at 12 09 59

Margin top should be 40px here instead of 30 px as per https://app.zeplin.io/project.html#pid=57adc4aaa7cbe69d4ad412a7&sid=57b6eead259157c43470dd7c

**Comment by AndrewBestbier** [comment]
screen shot 2016-08-23 at 12 11 44
43px margin-top 

(Sorry @lpil I know css is the worst)

**Pull request merged**